src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/products/features.yml

```diff
@@ -4,6 +4,7 @@ admin_feature_get_feature_values:
   defaults:
     idFeature: 0
     _controller: PrestaShopBundle\Controller\Admin\FeatureController::getFeatureValuesAction
+    _legacy_controller: AdminFeatures
```

Comment:
```suggestion
    _legacy_controller: AdminProducts
```

Reply:
Actually, it's a "sub controller" of Product page, this API is only used in the product edition page (like warehouse and suppliers)

Reply:
By the way, I just noticed that this route is also used in product V2, I marked this controller as deprecated but if we remove it (for v9) the page won't work anymore So we should either:
- remove the deprecation notice from `PrestaShopBundle\Controller\Admin\FeatureController`
- copy this action into the proper controller `PrestaShopBundle\Controller\Admin\Sell\Catalog\FeatureController` with a new proper route and update the [JS code of product page v2 here](https://github.com/PrestaShop/PrestaShop/blob/8.1.x/admin-dev/themes/new-theme/js/pages/product/edit/manager/feature-values-manager.ts#L145)